include/poller.php

```diff
@@ -20,14 +20,16 @@
 function poller_run($argv, $argc){
 	global $a, $db, $poller_up_start, $poller_db_duration;
 
-	$poller_up_start = microtime(true);
-
-	$a = new App(dirname(__DIR__));
+	if (is_null($a)) {
```

Comment:
it seems as if here you had reverted some programming as well. I guess this wasn't done intentionally?